README.md

```diff
@@ -93,6 +93,7 @@ Using unattended\_upgrades simply consists of including the module and if needed
 * `minimal_steps` (`true`): Split the upgrade process into sections to allow shutdown during upgrade.
 * `origins`: The repositories from which to automatically upgrade included packages.
 * `package_ensure` (`installed`): The ensure state for the 'unattended-upgrades' package.
+* `random_sleep` (`300`): Seconds of delay on the cron job execution.
```

Comment:
I just realized that this is potentially misleading. The randomness can be gleaned from the option name but the description should be phrased better.

Not a blocker; merging now and we'll fix that in post :-)